sos/plugins/tomcat.py

```diff
@@ -24,6 +24,10 @@ class Tomcat(Plugin, RedHatPlugin):
 
     packages = ('tomcat6', 'tomcat')
 
+    option_list = [
+        ("log", "gathers all tomcat logs", "slow", False)
+    ]
+
```

Comment:
Nack. There is a global `all_logs` option that is wired to the `--all-logs` command line switch. All plugins should use this rather than a plugin-specific option.

You can look at other plugins in the current tree for examples or take a look at the [plugin options](https://github.com/sosreport/sos/wiki/Plugin-options) wiki entry.